azerothcore/azerothcore-wotlk
# refactor(DB): Adjust table/column names to consistent standards.

## Changes Proposed:
- Adjusts table/column names in auth, characteres, and world DBs to [recent SQL standards](https://www.azerothcore.org/wiki/sql-standards#naming-of-tables-and-columns), as well as subjectively improving various column names.

## Issues Addressed:

- Closes #11005

## SOURCE:

N/A

## Tests Performed:

- Tested each query individually to make sure no errors occurred on my local DBs.

## How to Test the Changes:

1. Run the queries.
2. Observe that the auth/worldservers load properly and nothing changes in-game.

## Known Issues and TODO List:

- This is obviously not done. I'm opening this now so that any feedback on table/column name changes have time to be seen by contributors/reviewers/maintainers alike, and any changes that should be made can be made. The SQL queries don't have any work left to do on my end unless changes are requested, all that's left is adjusting the names in queries within the core.
- [X] Auth DB rename query.
- [X] Characters DB rename query.
- [X] World DB rename query.
- [X] Auth DB query core adjustments.
- [x] Characters DB query core adjustments.
- [ ] World DB query core adjustments.


## How to Test AzerothCore PRs
 
When a PR is ready to be tested, it will be marked as **[WAITING TO BE TESTED]**.

You can help by testing PRs and writing your feedback here on the PR's page on GitHub. Follow the instructions here:

http://www.azerothcore.org/wiki/How-to-test-a-PR

**REMEMBER**: when testing a PR that changes something **generic** (i.e. a part of code that handles more than one specific thing), the tester should not only check that the PR does its job (e.g. fixing spell XXX) but **especially** check that the PR does not cause any regression (i.e. introducing new bugs).

**For example**: if a PR fixes spell X by changing a part of code that handles spells X, Y, and Z, we should not only test X, but **we should test Y and Z as well**.
